python/torch_mlir/dialects/torch/importer/jit_ir/csrc/ivalue_importer.cpp

```diff
@@ -11,6 +11,7 @@
 #include "class_annotator.h"
 #include "function_importer.h"
 #include "torch_to_mlir_utils.h"
+#include "packed_params.h"
```

Comment:
this header in pytorch is only included if you build quantized infra [quantized/cpu/packed_params.h](https://github.com/pytorch/pytorch/blob/1f29b3130af218847a043e58fdc64511bbe072fe/aten/src/ATen/native/quantized/cpu/packed_params.h) even though from the looks of it there's nothing quantized about it.